sdk/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This changelog summarizes major changes between GraalVM SDK versions. The main f
 
 ## Version 23.1.5
 * (GR-54674) Added the options `engine.TraceSourceCache` and `engine.TraceSourceCacheDetails` to allow tracing of source cache hits, misses, failures and evictions.
+## Version 25.0.0
+* GR-60636 Truffle now stops compiling when the code cache fills up on HotSpot. A warning is printed when that happens.
+* GR-51664 Improved `PolyglotException#toString` and `PolyglotException#printStackTrace`.
+  * The short description returned by `PolyglotException#toString` now starts with the qualified name of the metaobject of the guest exception, if the exception represents a guest exception that has a metaobject. Otherwise, it starts with the qualified name of the `PolyglotException` class.
+  * `PolyglotException#printStackTrace` now always starts with the string returned by `PolyglotException#toString()` like for regular Java Throwable objects.
+* GR-54673 Added the option `engine.MaximumCompilations` to protect against too many repeated compilations of the same call target. The default value is `100`.
```

Comment:
This should either be removed or moved at the top under `## Version 23.1.8` keeping only the GR-54673 addition.